frontend/screens/Analytics/components/MyGraphs/AllGraphs.js

```diff
@@ -0,0 +1,177 @@
+//AllGraphs.js
```

Comment:
This file can be refactored as the only input will change for getting the graphs for respective social handle, so we should be able to reuse code instead of defining 3 constants?